app/gateway/changelog.md

```diff
@@ -6,6 +6,282 @@ no_version: true
+
+### Features
+
+#### Core
+
+* When `router_flavor` is `traditional_compatible`, Kong Gateway verifies routes created 
+  using the expression router instead of the traditional router to ensure created routes
+  are compatible.
+  [#9987](https://github.com/Kong/kong/pull/9987)
+* In DB-less mode, the `/config` API endpoint can now flatten all schema validation
+  errors into a single array using the optional `flatten_errors` query parameter.
+  [#10161](https://github.com/Kong/kong/pull/10161)
+* The upstream entity now has a new load balancing algorithm option: [`latency`](/gateway/latest/how-kong-works/load-balancing/#balancing-algorithms).
+  This algorithm chooses a target based on the response latency of each target
+  from prior requests.
+  [#9787](https://github.com/Kong/kong/pull/9787)
+* The Nginx `charset` directive can now be configured with Nginx directive injections.
+    Set it in Kong Gateway's configuration with [`nginx_http_charset`](/gateway/latest/reference/configuration/#nginx_http_charset)
+    [#10111](https://github.com/Kong/kong/pull/10111)
+* The services upstream TLS configuration is now extended to the stream subsystem.
+  [#9947](https://github.com/Kong/kong/pull/9947)
+* Added the new configuration parameter [`ssl_session_cache_size`](/gateway/latest/reference/configuration/#ssl_session_cache_size), 
+which lets you set the Nginx directive `ssl_session_cache`.
+  This configuration parameter defaults to `10m`.
+  
+  Thanks [Michael Kotten](https://github.com/michbeck100) for contributing this change.
+  [#10021](https://github.com/Kong/kong/pull/10021)
+
+#### Enterprise
+
+* Added two debugging endpoints to the Admin API:
+    * `/debug/profiling/cpu`: Instruction-based and timer-based Lua VM CPU profiling.
+    * `/debug/profiling/gc-snapshot`: Lua GC heap snapshot.
+
+##### Kong Manager
+
+* Improved the editor for expression fields. Any fields using the expression router now have syntax highlighting, autocomplete, and route validation.
+* Improved audit logs by adding `rbac_user_name` and `request_source`. 
+By combining the data in the new `request_source` field with the `path` field, you can now determine login and logout events from the logs.
+* License information can now be copied or downloaded into a file from Kong Manager. 
+* Kong Manager now supports the `POST` method for OIDC-based authentication.
+* Keys and key sets can now be configured in Kong Manager.
+* Optimized the color scheme for `http` method badges.
+
+#### Plugins
+
+* **Plugin entity**: Added an optional `instance_name` field, which identifies a
+  particular plugin entity.
+  [#10077](https://github.com/Kong/kong/pull/10077)
+
+* [**New plugin: Datadog Tracing**](/hub/kong-inc/datadog-tracing) (`datadog-tracing`)
+  * The Datadog Tracing plugin integrates Kong Gateway with the Datadog Application Performance Monitoring (APM) platform so that proxy requests handled by Kong Gateway can be identified and analyzed in Datadog.
```

Comment:
```suggestion
  * The Datadog Tracing plugin integrates Kong Gateway with the Datadog Application Performance Monitoring (APM) platform so that proxy requests handled by Kong Gateway can be identified and analyzed by Datadog.
```